python/dataproc_templates/elasticsearch/README.md

```diff
@@ -0,0 +1,957 @@
+## Elasticsearch to Cloud Storage
+
+Template for exporting an Elasticsearch Index to files in Google Cloud Storage. 
+
+It supports writing JSON, CSV, Parquet and Avro formats.
+
+It uses the [Elasticsearch Spark Connector](https://www.elastic.co/guide/en/elasticsearch/hadoop/current/index.html) for reading data from Elasticsearch Index.
+
+The template can support the Elasticsearch versions >= 7.12.0, using the appropriate Elasticsearch Spark Connector
+
+## Arguments
+- `es.gcs.input.node`: Elasticsearch Node Uri (format: mynode:9600)
+- `es.gcs.input.index`: Elasticsearch Input Index Name (format: <index>/<type>)
+- `es.gcs.input.user`: Elasticsearch Username
+- `es.gcs.input.password`: Elasticsearch Password
+- `es.gcs.output.format`: Cloud Storage Output File Format (one of: avro,parquet,csv,json)
+- `es.gcs.output.location`: Cloud Storage Location to put Output Files (format: `gs://BUCKET/...`)
+- `es.gcs.output.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists) (Defaults to append)
+
+#### Optional Arguments
+
+- `es.gcs.input.es.nodes.path.prefix`: Prefix to add to all requests made to Elasticsearch
+- `es.gcs.input.es.query`: Holds the query used for reading data from the specified Index
+- `es.gcs.input.es.output.json`: Whether the output from the connector should be in JSON format or not (default true)
```

Comment:
Is this used as an intermediate output format? What is user provides this as False?

Reply:
May be rename it to something like `connector.output` instead of `output` to avoid confusion with `es.gcs.output.format`, if that makes sense?

Reply:
Hardcoded the value of `es.output.json` to true, as in order to run the spark transformations the output from the ES is required to be in JSON format.